_articles/organizations/getting-started-organizations.md

```diff
@@ -0,0 +1,185 @@
+  * [What are organizations?](#what-are-organizations-)
+- [Set up the First and Second User Accounts](#set-up-the-first-and-second-user-accounts)
+  * [1. Create an account for User 1](#1-create-an-account-for-user-1)
+  * [2. Verify Your Email](#2-verify-your-email)
+  * [3. Set up the Second User Account](#3-set-up-the-second-user-account)
+- [Setup and Share with Organizations](#setup-and-share-with-organizations)
+  * [4. Create a new Organization](#4-create-a-new-organization)
+  * [5. Add the Second User to your Organization](#5-add-the-second-user-to-your-organization)
+  * [6. Accepting the Invitation](#6-accepting-the-invitation)
+  * [7. Sharing within an Organization](#7-sharing-within-an-organization)
+- [Continuing with Bitwarden](#continuing-with-bitwarden)
+# Introduction
+Using a password manager makes it easy to store and access unique and secure passwords across all of your devices. This keeps your online accounts safer than repeating passwords or leaving them exposed in unencrypted formats like spreadsheets, documents, or sticky notes!
+
+To get even more out of your password manager, you can collaborate and share with others. At work, you may choose to share common logins to office wifi networks and online services, or a shared company credit card, or a set of secure notes. Personally you may choose to share a few passwords with a significant other.
+
+Bitwarden enables all types of password and secure sharing through Organizations. 
+
+If you would like to try it out, Bitwarden has a free account for individuals, and the ability to set up a free Organization for up to two users and two Collections of secure information.
+
+This tutorial walks you through the setup for two basic accounts and sharing in an Organization.
+## What are organizations?
+A Bitwarden organization is an entity that relates users together that want to share items. An organization could be a family, team, company, or any other type of group that desires to share items in Bitwarden.
+
+For more, please see [What are organizations?](https://help.bitwarden.com/article/what-is-an-organization/)
+
+**Are Organizations Different than a Premium Membership?**
+Yes. Organizations are for sharing items across multiple users
+Premium membership gives your individual user account additional premium features, including
+1GB encrypted file storage
+Two-step login with YubiKey, FIDO U2F, & Duo
+Password hygiene & vault health reports
+TOTP authenticator key storage & code gen.
+Priority customer support
+# Set up the First and Second User Accounts
+You can choose many ways to create an account at [www.bitwarden.com](www.bitwarden.com) where you will find
+Desktop Apps for Windows, Mac, and Linux
+Browser Extensions for Google Chrome, Mozilla Firefox, Opera, Microsoft Edge, Safari, Vivaldi, Brave, Tor Browser
+Mobile Apps for iOS and Android
+Command Line Tools
+Web access through [https://vault.bitwarden.com](https://vault.bitwarden.com)
```

Comment:
another list?

Reply:
just replicating the list of Premium features from bitwarden.com